blivet/formats/fs.py

```diff
@@ -77,7 +77,8 @@ class FS(DeviceFormat):
     # value is already unpredictable and can change in the future...
     _metadata_size_factor = 1.0
 
-    config_actions_map = {"label" : "write_label"}
+    config_actions_map = {"label": "write_label",
+                          "mountpoint": None}
```

Comment:
This is intended to change (future) mountpoint for existing devices during installation or some "installation mode". It indeed is quite a weird action because it does nothing during `execute`, it just set the attribute.